packages/wrapt/test_wrapt.py

```diff
@@ -54,8 +54,8 @@ def test_argspec(self):
             self.assertEqual(function1o_argspec, function1d_argspec)
 
         def test_getmembers(self):
-            function1o_members = inspect.getmembers(function1o)
-            function1d_members = inspect.getmembers(function1d)
+            inspect.getmembers(function1o)
+            inspect.getmembers(function1d)
```

Comment:
Not sure what these were for

Reply:
I copied this from the wrapt test suite but then I deleted some lines that were causing trouble due to my crappy copy-paste job. It looks like `test_getmembers` doesn't do anything anymore, so we should probably remove it.